index.bs

```diff
@@ -270,19 +274,34 @@ the user agent <em>must</em> run the following steps:
 When <dfn>parsing an animator timeline list</dfn> for {{animatorCtor}},
 the user agent <em>must</em> run the following steps:
 
-    1. Let |properties| be an empty <code>sequence&lt;DOMString></code>.
+    1. Let |timelines| be an empty <code>sequence&lt;Object></code>.
 
     2. Let |timelinesIterable| be the result of <a>Get</a>(|animatorCtor|, |timelines|).
 
-    3. For each |timeline| in |timelinesIterable|, run the following steps:
+    3. For each |timeline| in |timelinesIterable|, run the following sub-steps:
 
-        - If |timeline| is not "document" or "scroll", <a>throw</a> a <a>NotSupportedError</a>
-            and abort all these steps.
+        1. If <a>Type</a>(|timeline|) is {{DOMString}} and |timeline| is "document",
+             push a default {{DocumentTimelineOptions}} onto |timelines| and abort the following
+             sub-steps.
 
-    4. If |timelinesIterable| is not undefined, then set |timelines| to the result of
-        converting |timelinesIterable| to a <code>sequence&lt;DOMString></code>. If an
-        exception is thrown, rethrow the exception and abort all these steps.
+        2. If <a>Type</a>(|timeline|) is {{DOMString}} and |timeline| is "scroll",
+             push a default {{ScrollTimelineOptions}} onto |timelines| and abort the following
+             sub-steps.
```

Comment:
To avoid an additional rule per timeline type, if the type is {{DOMString}} we could set a variable to be {type: |passedInValue|, options: {}} and then proceed with the remaining steps.